sirplatform.mk

```diff
@@ -282,6 +282,23 @@ ifeq ($(AIXTLS),1)
   SIR_XFLAGS+=-fPIC -ftls-model=global-dynamic
 endif
 
+##############################################################################
+# CompCert (WIP)
```

Comment:
If it's WIP, why PR?  Also, forgot to mention it before, but if we're adding a preproc macro, we need to update README (iirc) and `docs/sources/config.md`

Reply:
I put WIP there because it needs the unreleased version from the vendors GitLab repo to build out of the box.  I'll add a better comment and update the documentation.

Reply:
Done.